lib/travis/api/serialize/v2/http/build.rb

```diff
@@ -8,27 +8,27 @@ module Http
           class Build
             include Formats
 
-            attr_reader :build, :options
-
-            def initialize(build, options = {})
-              options[:include_jobs] = true unless options.key?(:include_jobs)
```

Comment:
Per conversation with @svenfuchs I've removed the conditional behavior around `:include_jobs` since it appears nothing is (currently) explicitly setting it to `false`.